src/pcedit/pc.graphics.cpp

```diff
@@ -29,7 +29,8 @@ extern fs::path file_in_mem;
 extern short store_flags[3];
 extern short current_active_pc;
 
-rectangle whole_win_rect = {0,0,440,590};
+// TODO why are width and height switched here?
```

Comment:
What do you mean, they're switched? The rectangle constructor is top, left, bottom, right.

Reply:
I'm really used to the order being {x, y, width, height} in other frameworks. My bad.